content/en/agent/basic_agent_usage/aix.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Basic Agent Usage for AIX
+kind: documentation
+further_reading:
+---
+
+<div class="alert alert-info">
+The Datadog UNIX agent is being developed for specific system architectures, and is not the same as Agents 5 and 6.
+</div>
+
+This page outlines the installation and configuration of the Datadog UNIX Agent for AIX.
+
+**Note:** The Datadog UNIX Agent supports AIX 6.1, 7.1 and 7.2.
+
+## Installation
+
+Download links for the latest releases can be found on [this page][1].
+
+The installer may be executed as follows (as root):
+
+```shell
```

Comment:
We probably want to add mention of the single-step install script:

```
DD_API_KEY=<somekey> ksh -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-unix-agent/master/scripts/install_script.sh)"
```

and mention it's a kprn shell script.

I'm going to improve proxy support on the script by GA.

Reply:
Added a link to the in-app download page instead, which would be better to redirect users to.